client/consensus/manual-seal/src/heartbeat_stream.rs

```diff
@@ -0,0 +1,133 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! HeartbeatStream type and HeartbeatOptions used in instant seal
+use std::{pin::Pin};
+use futures::{
+	prelude::*,
+	task::{Context, Poll},
+};
+use tokio::time::{Duration, Instant, Delay};
+
+use crate::rpc::{EngineCommand};
+
+/// Constants for generating default heartbeat options
+const DEFAULT_MAX_BLOCKTIME: u64 = 30;
```

Comment:
Do we really need them as constants?

Reply:
They won't be used anywhere else outside of this file. Should they just be variables? Or just hard-coded into the `new` function?

Reply:
I've moved them to the `default` function.